CHANGELOG.rst

```diff
@@ -2,10 +2,12 @@ CHANGELOG
 =========
 
 0.3.0 (2024-11-29)
-------------------
+---------------------
 
 **Breaking changes**
 
+* Simplify the project structure and debugging by @tatiana in #93
```

Comment:
does #93 contain a link to the issue or PR or just rendered as text?

Reply:
ATM is just rendered as text - it is a sort of improvement from what we had before, but I agree, it would be better with proper links - I'll add them

Reply:
Addressed in 18974b41a6d2a1791774f8cf09e1f6da320148b4